### PR TITLE
Adopt new changes BS5

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Accordion.php [hunk 1]
```diff
@@ -192,6 +221,22 @@ public function options(array $value): self
         return $new;
     }
 
+    /**
+     * Add to remove the default, some borders, and some rounded corners to render accordions edge-to-edge with their
+     * parent container.
```
Comment (samdark) [inline src/Accordion.php:226]:
```suggestion
     * Add to remove the default background-color, some borders, and some rounded corners
     * to render accordions edge-to-edge with their parent container.
```

File: src/Accordion.php [hunk 2]
```diff
@@ -192,6 +221,22 @@ public function options(array $value): self
...
+     *
+     * @return $this
```
Comment (samdark) [inline src/Accordion.php:228]:
```suggestion
     * @return self
```

File: src/NavBar.php [hunk 3]
```diff
@@ -252,17 +229,19 @@ public function brandImage(string $value): self
     }
 
     /**
-     * The URL for the brand's hyperlink tag and will be used for the "href" attribute of the brand link. Default value
-     * is '/' will be used. You may set it to `null` if you want to have no link at all.
+     * The Link for the brand's hyperlink tag and will be used for the "href" attribute of the brand link. Default value
+     * is '/' will be used. You may set it to `` if you want to have no link at all.
```
Comment (samdark) [inline src/NavBar.php:233]:
```suggestion
     * The URL for the brand's hyperlink tag and will be used for the "href" attribute of the brand link. Default value
     * is "/". You may set it to empty string if you want no link at all.
```

File: src/NavBar.php [hunk 4]
```diff
@@ -252,17 +229,19 @@ public function brandImage(string $value): self
...
      *
      * @param string $value
      *
      * @return $this
+     *
+     * @link https://getbootstrap.com/docs/5.0/components/navbar/#text
      */
-    public function brandUrl(string $value): self
+    public function brandLink(string $value): self
```
Comment (samdark) [inline src/NavBar.php:241]: I think it's still URL.

File: tests/AssetTest.php [hunk 5]
```diff
@@ -35,8 +35,6 @@ public function registerDataProvider(): array
      */
     public function testAssetRegister(string $type, string $asset, ?string $depend = null): void
```
Comment (samdark) [inline tests/AssetTest.php:36]: This test wasn't testing anything useful (it was testing that asset manager works and that's already tested) so I've removed it from master.

What is worth testing is the fact that widgets, when used, are registering expected assets.

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Accordion.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Accordion.php:226]: done.

File: src/Accordion.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Accordion.php:228]: done.

File: src/NavBar.php (hunk 3 above)
Comment (terabytesoftw) [reply src/NavBar.php:233]: done.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**